test/vanilla/version-tolerant/Expected/AcceptanceTests/MediaTypesVersionTolerant/mediatypesversiontolerant/_operations/_operations.py

```diff
@@ -441,55 +225,12 @@ def binary_body_with_two_content_types(self, message: IO, **kwargs: Any) -> str:
         return cast(str, deserialized)
 
     @distributed_trace
-    def binary_body_with_three_content_types(self, message: IO, **kwargs: Any) -> str:
-        """Binary body with three content types. Pass in string 'hello, world' with content type
-        'text/plain', {'hello': world'} with content type 'application/json' and a byte string for
-        'application/octet-stream'.
+    @abc.abstractmethod
+    def binary_body_with_three_content_types(self, *args, **kwargs) -> str:
```

Comment:
Do we need to generate as abstract while the body is in binary schema?

Reply:
Not 100% sure what you mean, but basically if we find the body can be three different kinds of bodies (in this case a streamed body, a JSON body, or a text body), then we mark it as abstract